components/alert/alert.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alert • PowertrainUI</title>
+  </head>
+  <body>
+    <main class="wrapper">
+      <nav class="wrapper__nav-top">
+        <div class="nav__logo">
+          <a href="#">
+            <h2 class="nav__title">PowertrainUI</h2>
+            <img
+              class="nav__img"
+              src="./images/icon.png"
+              alt="PowertrainUI logo"
+            />
+          </a>
+        </div>
+        <ul class="nav__menu">
+          <li class="nav__menu-item"><a href="#">Get Started</a></li>
```

Comment:
Looks like you are yet to add the links to your anchor tag. 

Add it first then merge the branch.

Reply:
Will add anchor tag links when working on doc html